src/modules/shortcut_guide/overlay_window.cpp

```diff
@@ -585,7 +605,7 @@ void D2DOverlayWindow::render(ID2D1DeviceContext5* d2d_dc) {
     }
     // If the animation is done show the thumbnail
     //   we cannot animate the thumbnail, the animation lags behind
-    minature_shown = show_thumbnail(thumbnail_pos, current_anim_value);
+    minature_shown = show_thumbnail(thumbnail_pos, 1.0f);
```

Comment:
Can you refactor the second parameter out of the `show_thumbnail()`? Looks like `SetLayeredWindowAttributes` call also fades the thumbnail preview.

Reply:
Do you mean completely take out the `alpha` variable out of `show_thumbnail()`? I did that and it made the thumbnail preview look darker.

Reply:
Just hardcode ` thumb_properties.opacity = 255;` in `D2DOverlayWindow::show_thumbnail` and remove the alpha parameter. It should look the same.

Reply:
Done https://github.com/microsoft/PowerToys/pull/521/commits/f53f9480b7513371f431a85c22486e3105fcad85